Marshal/UnmarshalUpdating.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  M A R S H A L
+//
+//       ()
+//       /\
+//  ()--'  '--()
+//    `.    .'
+//     / .. \
+//    ()'  '()
+//
+//
+
+
+import Foundation
+
+
+public protocol UnmarshalUpdating {
```

Comment:
I actually have noticed this and it has been driving me nuts. Saving it for a different commit or PR